src/PluginMediaStreamTrack.swift

```diff
@@ -61,8 +65,13 @@ class PluginMediaStreamTrack : NSObject {
 		_ eventListener: @escaping (_ data: NSDictionary) -> Void,
 		eventListenerForEnded: @escaping () -> Void
 	) {
-		NSLog("PluginMediaStreamTrack#setListener() [kind:%@, id:%@]", String(self.kind), String(self.id))
-
+        if(self.eventListener != nil){
+            NSLog("PluginMediaStreamTrack#setListener():Error Listener already Set [kind:%@, id:%@]", String(self.kind), String(self.id));
+            return;
+        }
```

Comment:
LGTM

Reply:
@hthetiot what is LGTM and LGMTL?

Reply:
LGTM means Looks Good To Me

Reply:
Yes LGTM "Looks Good To Me" mean code has been reviewed and changes is valid it's a conventional way to review code :)